bin/macro-benchmark.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+let fs = require('fs-extra');
+let path = require('path');
+
+let args = process.argv;
+let [ command, ...opts ] = args.slice(2);
+
+let commands = { "add-config": addConfig };
+
+let cmd;
+if (cmd = commands[command]) {
+  cmd(opts);
+} else {
+  throw new Error('Command not specified.');
+}
+
+function addConfig(options) {
+  let [ name ] = options;
+  let cwd = process.cwd();
+
+  if (fs.existsSync(path.join(cwd, 'config'))) {
+    let archivePath = path.join(cwd, 'config', 'archive', '.gitkeep');
+    fs.outputFileSync(archivePath, '');
+
+    let perProjectConfigFile = path.join(cwd, 'config', 'index.js');
+    let perProjectConfig = `
+      module.exports = {
```

Comment:
I believe as written this will create a `module.exports` with ~ 8 leading spaces, which is probably not what we want to do.

To address that we should bring in [common-tags](https://github.com/declandewet/common-tags) and use `stripIndent` tagged template literal:

```js
     stripIndent`
        // this will start without any padding
     `
```